tests/Wallabag/CoreBundle/Controller/EntryControllerTest.php

```diff
@@ -977,7 +978,7 @@ public function testNewEntryWithDownloadImagesEnabled()
         $this->logInAs('admin');
         $client = $this->getClient();
 
-        $url = 'http://www.20minutes.fr/montpellier/1952003-20161030-video-car-tombe-panne-rugbymen-perpignan-improvisent-melee-route';
+        $url = self::An_URL_CONTAINING_AN_ARTICLE_WITH_IMAGE;
```

Comment:
Graby returned the article without image so I use an other page with an image

Reply:
Looks like 20minutes is badly parsed now 😕